Source/DafnyLanguageServer/Handlers/DafnyCompletionHandler.cs

```diff
@@ -122,6 +199,15 @@ private static bool IsConstructor(ILegacySymbol symbol) {
         return symbol is MethodSymbol method && method.Name == "_ctor";
       }
 
+      private CompletionItem CreateCompletionItem(string attributeName) {
+        return new CompletionItem {
+          Label = attributeName,
+          Kind = CompletionItemKind.Constructor,
+          InsertText = attributeName, // TODO: Parentheses
+          Detail = "" // TODO: Details of attribute name
```

Comment:
These look more like unfinished PR todos rather than planned improvements for the next iteration